amgut/lib/data_access/sql_connection.py

```diff
@@ -1,72 +1,204 @@
-from __future__ import division
-
 # -----------------------------------------------------------------------------
-# Copyright (c) 2014--, The American Gut Development Team.
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
I would be OK with that if we could somehow guarantee that we will not
be modifying this - it's subtle, but it could lead to annoyances in the
future. Maybe worth it to include in the License file that we are
including this file as part of the software? We do this in emperor,
skbio and AFAIK others.

On (Oct-02-15|15:31), Jose Navas wrote:
> >  # -----------------------------------------------------------------------------
> > -# Copyright (c) 2014--, The American Gut Development Team.
> > +# Copyright (c) 2014--, The Qiita Development Team.
> 
> This code is copy pasted from qiita - I think it is ok leaving this copyright note.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/american-gut-web/pull/519/files#r41074653